perfTest/run.sh

```diff
@@ -7,7 +7,7 @@ cd "${0%/*}"
 export NO_LOG_SUCCESS=1
 
 # if connection string not provided, assume postgres is available locally
-export PERF_DATABASE_URL=${TEST_CONNECTION_STRING-graphile_worker_perftest}
+export PERF_DATABASE_URL=${TEST_CONNECTION_STRING}-graphile_worker_perftest
```

Comment:
This isn't the intention of this code; it's using bash parameter substitution to apply a default value if TEST_CONNECTION_STRING is not set. It seems, however, that TEST_CONNECTION_STRING is set, and that it's in use by something else so cannot be reset? I'll have to look into this more; I believe this was caused by #64

Reply:
Ahh - Okay! I will close this PR then.

Reply:
Thanks anyway!